ipyaggrid/util.py

```diff
@@ -1,6 +1,5 @@
 import os
 import re
-import collections
```

Comment:
We don't want unrelated changes in our commits/PRs. Removing a previously unused import is not necessary for this feature. We want to fix all code quality issues at once in the PR that also introduces the tools to check them.

Reply:
Done, although I don't really see why this would ever be a controversial thing to do.